packages/babel-preset-react/src/normalize-options.js

```diff
@@ -0,0 +1,100 @@
+import { OptionValidator } from "@babel/helper-validator-option";
+const v = new OptionValidator("@babel/preset-react");
+
+export default function normalizeOptions(options = {}) {
+  if (process.env.BABEL_8_BREAKING) {
+    if ("useSpread" in options) {
+      throw new Error(
+        '@babel/preset-react: Since Babel 8, an inline object with spread elements is always used, and the "useSpread" option is no longer available. Please remove it from your config.',
+      );
+    }
+
+    if ("useBuiltIns" in options) {
+      const useBuiltInsFormatted = JSON.stringify(options.useBuiltIns);
+      throw new Error(
+        `@babel/preset-react: Since "useBuiltIns" is removed in Babel 8, you can remove it from the config.
+- Babel 8 now transforms JSX spread to object spread. If you need to transpile object spread with
+\`useBuiltIns: ${useBuiltInsFormatted}\`, you can use the following config
+{
+  "plugins": [
+    ["@babel/plugin-proposal-object-rest-spread", { "loose": true, "useBuiltIns": ${useBuiltInsFormatted} }]
+  ],
+  "presets": ["@babel/preset-react"]
+}`,
+      );
+    }
+
+    const TopLevelOptions = {
+      development: "development",
+      importSource: "importSource",
+      pragma: "pragma",
+      pragmaFrag: "pragmaFrag",
+      pure: "pure",
+      runtime: "runtime",
+      throwIfNamespace: "throwIfNamespace",
+    };
+    v.validateTopLevelOptions(options, TopLevelOptions);
+    const development = v.validateBooleanOption(
+      TopLevelOptions.development,
+      options.development,
```

Comment:
I think we need a default value of false here via https://github.com/babel/babel/pull/12741/files#diff-98ee15bc5afa1493a44db1f236a831833828577f9ef8034cc6a0a9ac582f389cL10

```
let { pragma, pragmaFrag, development = false } = opts;
```